x-pack/plugins/lens/public/indexpattern_datasource/operations/definitions/ranges/constants.ts

```diff
@@ -0,0 +1,20 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+export const TYPING_DEBOUNCE_TIME = 256;
+// Taken from the Visualize editor
+export const FROM_PLACEHOLDER = '\u2212\u221E';
+export const TO_PLACEHOLDER = '+\u221E';
+
+export const DEFAULT_INTERVAL = 1000;
+export const AUTO_BARS = 'auto';
+export const MIN_HISTOGRAM_BARS = 1;
+export const SLICES = 6;
```

Comment:
My opinion on the granularity settings has changed a bit since we last talked about it. The main reason is that with these settings, there is a big jump from 1 bar to 10-16 bars, which prevents me from getting anything in between. I'm not sure what the right tweak is, but if I could hardcode my "optimal" steps it might be something like `[1, 5, 10, 20, 40, 80, 100, 200]`.

Reply:
I have some concerns about an hardcoded set of values:
1) we might have to think about a strategy in case the user specify a `maxHistogramBars` value which is not covered in the list
2) because the range input starts with the middle mark, the list might have an odd number of values

Because of 1) I feel we may fall again under a slices approach - maybe it's a rare scenario ok -, while for 2) it's more a stylistic reason.